source/use-cases/sensitive-data-encryption.txt

```diff
@@ -24,54 +24,46 @@ MongoDB offers several methods that protect your data from unauthorized access i
+In this scenario, we secure sensitive data on a Medical Care Management System which stores patients' personal information, insurance information, and medical records for a fictional company, *MedcoMD*. None of the patient data is public, and certain data such as their social security number (SSN), insurance policy number, and vital sign measurements are particularly sensitive and subject to privacy compliance. It is important for the company and the patient that the data is kept private and secure.
 
 MedcoMD needs this system to satisfy the following use cases:
 
-- Doctors use the system to access Patients' medical records and add new vital sign measurements.
-- Receptionists use the system to schedule appointments for Patients, who are identified by a combination of contact information and their SSN.
-- Receptionists cannot access nor update Patients' medical records.
+- Doctors use the system to access Patients' medical records, insurance information, and add new vital sign measurements.
+- Receptionists use the system to verify the Patients' identity, using a combination of their contact information and last four digits of their Social Security Number (SSN).
+- Receptionists can view a Patient's insurance policy provider, but not the number
+- Receptionists cannot access a Patient's medical records
 
 MedcoMD is also concerned with disclosure of private information through any of the following methods:
 
-- Printing the information or leaving it displayed on a screen where unauthorized people can view it
-- Communicating on an insecure network
-- Gaining access to the database on one of the permissioned users
+- Accidental disclosure of data on the Receptionist's publicly-viewable screen
+- Communication of the data over an insecure network
+- Direct access to the database by a superuser such as a database administrator
 
 What can MedcoMD do to balance the functionality and access restrictions of their Medical Care Management System?
 
 Solution
 ~~~~~~~~
 
-The MedcoMD software engineers review the Medical Care Management System specification and determine that Doctors and Receptionists require different levels of access to the data.
+The MedcoMD software engineers review the Medical Care Management System specification and determine they must decide the proper method of limiting access to the sensitive data.
 
+The first MongoDB security feature they evaluated was `Encryption at Rest <https://docs.mongodb.com/manual/core/security-encryption-at-rest/>`_ which encrypts data at the storage level. However, this solution did not address the potential issue of database administrators or Receptionists viewing or exposing the sensitive data.
```

Comment:
Should we add a blurb to the second sentence that tells what it _does_ accomplish? Or do you think we shouldn't focus too much on it?

e.g. "This would prevent \<foo\>, but it does not address..."